[DOC] Improve contributors doc about mxgraph integration (#1337)
process-analytics/bpmn-visualization-js: docs/contributors/mxgraph-integration.md

Check the [GitHub Pull Request #291](https://github.com/process-analytics/bpmn-visualization-js/pull/291#issuecomment-642024601)
to see various positioning methods in action.

## Overlays

> We are hacking mxCellOverlays which originally only supports image shape. This let us use custom shapes.

Reviewer: ```suggestion
We are hacking mxCellOverlays which originally only supports image shape. This lets us use custom shapes.
```

Customization main points
- `MxGraphCustomOverlay`: add extra configuration and behavior to mxCellOverlay
- customized `mxCellRenderer`: change the overlays rendering code to transform `MxGraphCustomOverlay` into specific shapes 
according to its configuration.

More details are available in [#955](https://github.com/process-analytics/bpmn-visualization-js/issues/955).

## Custom CSS classes added to svg node generated by mxGraph

> In `ShapeConfigurator`, we customize the mxShape rendering code to add extra CSS classes depending on BPMN Elements kind.

Reviewer: ```suggestion
In `ShapeConfigurator`, we customize the mxShape rendering code to add extra CSS classes depending on BPMN Element kinds.
```

> This let managing styling with external CSS classes and is used for elements identification and selection.

Reviewer: ```suggestion
This lets managing the styling with external CSS classes and is used for element identification and selection.
```

Author: ```suggestion
It allows styling to be managed with external CSS classes and it is also used for element identification and selection.
```
